utils/constant.ts

```diff
@@ -0,0 +1,7 @@
+/* 정규표현식 */
+const REG_EXP = {
```

Comment:
아래처럼 바로 내보내기 하시는 것도 좋아보이네요!

```suggestion
export const REG_EXP = {
```